addons/website_livechat/static/src/legacy/public_livechat.scss

```diff
@@ -5,3 +5,24 @@
         right: $o-we-sidebar-width !important;
     }
 }
+
+.o_website_livechat_button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: filter 0.3s;
```

Comment:
@bvr-odoo Can you check to remove this delay as per PDE suggestion on the PR so we can r+ asap ?

Reply:
@rdeodoo ok done